src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -524,6 +546,14 @@ def makeDagSpecs(self, task, sitead, siteinfo, jobgroup, block, availablesites,
         dagSpecs = []
         i = startjobid
         temp_dest, dest = makeLFNPrefixes(task)
+        if task['tm_publication'] :
+            try:
+                validateLFNs(dest,outfiles)
+            except Exception as ex:
```

Comment:
Why not catch only the AssertionError? Really doesn't make a big difference but I see no reason to catch everything.